### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by anarro**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,10 +26,18 @@ RUN apt-get update && \
   chmod 755 /run.sh
 
 # Volumes
-VOLUME /var/lib/unifi-video /var/log/unifi-video
+VOLUME ["/var/lib/unifi-video"]
+VOLUME ["/var/log/unifi-video"]
+VOLUME ["/usr/lib/unifi-video/data/videos"]
```
Comment (chadhutchins182) [inline Dockerfile]: Why is this needed? Wouldn't the first line include any sub-directories which would include `["/usr/lib/unifi-video/data/videos"]`

Comment (anarro) [reply Dockerfile]: Hi,
Because my Doker don't show the current works dirs... and I read in a Docker docs thats correctly is with this format....

**Comment by fryfrog** [comment]
I think I may be the one that collapsed this down because it made for fewer build steps, but if this is the "right way", we should probably fix it.

**Comment by fryfrog** [comment]
I've re-done this w/ #61, making EXPOSE and VOLUME better, removing the log volume that goes away in 3.9.2.

**Comment by fryfrog** [comment]
This is done w/ 3.9.2 in a slightly different way due to loss of logs volume.
